lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -204,9 +204,9 @@
         <div *ngFor="let group of groupedProperties; let first = first" class="adf-metadata-grouped-properties-container">
             <mat-expansion-panel
                 [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                [expanded]="currentPanel.panelTitle === group.title && currentPanel.expanded"
-                (opened)="expandPanel(group.title)"
-                (closed)="closePanel(group.title)"
+                [expanded]="group.title === displayAspect || !displayDefaultProperties && first || currentPanel.panelTitle === group.title && group.expanded"
+                (opened)="group.expanded = true"
```

Comment:
This doesn't seem correct `expandPanel` and `closePanel` are both properly setting the `currentPanel` which we should rely on. Otherwise integration with other panels will be broken

Reply:
This breaks the case when we use `multi` property to allow multi accordion to be expanded at the same time.

Reply:
Okay in this case we need to run different methods/logic based on `multi` state we also have to make sure not to break regular use case without multi as in this case only one panel can be expanded at once

Reply:
addressed